packages/components/Menu/src/MenuItemRadio/MenuItemRadio.types.ts

```diff
@@ -0,0 +1,132 @@
+import type * as React from 'react';
+import type { ColorValue, ImageProps, ViewStyle } from 'react-native';
+
+import type { IViewProps } from '@fluentui-react-native/adapters';
+import type { IconPropsV1 as IconProps } from '@fluentui-react-native/icon';
+import type { PressablePropsExtended, PressableState } from '@fluentui-react-native/interactive-hooks';
+import type { TextProps } from '@fluentui-react-native/text';
+
+import type { MenuItemProps, MenuItemTokens } from '../MenuItem/MenuItem.types';
+
+export const menuItemRadioName = 'MenuItemRadio';
+
+export interface MenuItemRadioTokens
+  extends Omit<MenuItemTokens, 'submenuIndicatorPadding' | 'submenuIndicatorSize' | 'disabled' | 'focused' | 'hovered' | 'pressed'> {
+  /**
+   * Color of the icon
+   */
+  iconColor?: ColorValue;
+
+  /**
+   * Size of the icon. Pixels for SVG and points for font icon.
+   */
+  iconSize?: number;
+
+  /**
+   * Color of the background of the box containing the checkmark.
+   */
+  radioBackgroundColor?: ColorValue;
```

Comment:
All of the radio * props should be android/mobile only, right?

Reply:
yes will mark it @android only now.